NQuads.php

```diff
@@ -41,7 +41,11 @@ public function serialize(array $quads)
                     ? $quad->getObject()
                     : '<' . $quad->getObject() . '>';
             } else {
-                $result .= '"' . $quad->getObject()->getValue() . '"';
```

Comment:
Please change this to

```php
$search = array("\n", '"');
$replace = array('\n', '\"');
$result .= '"' . str_replace($search, $replace, $quad->getObject()->getValue()) . '"';
```

N-Quads supports `\n` for line-feeds, see https://www.w3.org/2013/N-QuadsTests/